relay/polkadot/src/lib.rs

```diff
@@ -537,6 +539,54 @@ parameter_types! {
 	pub const MaxNominations: u32 = <NposCompactSolution16 as frame_election_provider_support::NposSolution>::LIMIT as u32;
 }
 
+/// Custom version of `runtime_commong::era_payout` somewhat tailored for Polkadot's crowdloan
+/// unlock history. The only tweak should be
+///
+/// ```diff
+/// - let auction_proportion = Perquintill::from_rational(auctioned_slots.min(60), 200u64);
+/// + let auction_proportion = Perquintill::from_rational(auctioned_slots.min(60), 300u64);
+/// ```
+///
+/// See <https://forum.polkadot.network/t/adjusting-polkadots-ideal-staking-rate-calculation/3897>.
+fn polkadot_era_payout(
```

Comment:
I see that it is tweaked for Polkadot - but that means that we cannot test the logic on Kusama first?  
(not going to block, just an idea)

Reply:
This is already rolled out on Polkadot.

Reply:
Or better, it is approved and the runtime hits Polkadot in the next days.